src/pudl/package_data/eia923/column_maps/cooling_system_information.csv

```diff
@@ -0,0 +1,31 @@
+maximum_discharge_temperature_fahreneit,,,discharge_maximum_temperature,discharge_maximum_temperature,discharge_maximum_temperature_f,discharge_maximum_temperature_f,discharge_maximum_temperature_f,discharge_maximum_temperature_f,discharge_maximum_temperature_f,discharge_maximum_temperature_f,discharge_maximum_temperature_f,discharge_maximum_temperature_f,discharge_maximum_temperature_f,discharge_maximum_temperature_f,discharge_maximum_temperature_f
+temperature_method,,,method_for_temperatures,method_for_temperatures,method_for_temperatures,method_for_temperatures,method_for_temperatures,method_for_temperatures,method_for_temperatures,method_for_temperatures,method_for_temperatures,method_for_temperatures,method_for_temperatures,method_for_temperatures,method_for_temperatures
+diversion_volume_million_gallons,,,,,diversion_volume_million_gallons,diversion_volume_million_gallons,diversion_volume_million_gallons,diversion_volume_million_gallons,diversion_volume_million_gallons,diversion_volume_million_gallons,diversion_volume_million_gallons,diversion_volume_million_gallons,diversion_volume_million_gallons,diversion_volume_million_gallons,diversion_volume_million_gallons
+withdrawal_volume_million_gallons,,,,,withdrawal_volume_million_gallons,withdrawal_volume_million_gallons,withdrawal_volume_million_gallons,withdrawal_volume_million_gallons,withdrawal_volume_million_gallons,withdrawal_volume_million_gallons,withdrawal_volume_million_gallons,withdrawal_volume_million_gallons,withdrawal_volume_million_gallons,withdrawal_volume_million_gallons,withdrawal_volume_million_gallons
+discharge_volume_million_gallons,,,,,discharge_volume_million_gallons,discharge_volume_million_gallons,discharge_volume_million_gallons,discharge_volume_million_gallons,discharge_volume_million_gallons,discharge_volume_million_gallons,discharge_volume_million_gallons,discharge_volume_million_gallons,discharge_volume_million_gallons,discharge_volume_million_gallons,discharge_volume_million_gallons
+consumption_volume_million_gallons,,,,,consumption_volume_million_gallons,consumption_volume_million_gallons,consumption_volume_million_gallons,consumption_volume_million_gallons,consumption_volume_million_gallons,consumption_volume_million_gallons,consumption_volume_million_gallons,consumption_volume_million_gallons,consumption_volume_million_gallons,consumption_volume_million_gallons,consumption_volume_million_gallons
+chlorine_annual_thousand_lbs,annual_chlorine,annual_chlorine,,,,,,,,,,,,,
+annual_withdrawal_rate_tenth_cubic_feet_per_second,annual_withdrawal_rate,annual_withdrawal_rate,,,,,,,,,,,,,
```

Comment:
This is 0.1 cubic feet - how should we name this column?

Reply:
I don't think we should do any conversions in the extraction step. Definitely during the transform, but for now this is just a question of how to name a raw column.